src/components/Elements/index.js

```diff
@@ -0,0 +1,91 @@
+import { Center, Button, Flex } from "@chakra-ui/react";
+import { extendTheme } from "@chakra-ui/react";
+import { Text } from "@chakra-ui/react";
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { Link } from "react-router-dom";
+import Formfortrial from "../FormForTrial";
+
+const theme = extendTheme({
```

Comment:
This is not used by the `Banner` component so how about putting this in the `utils/theme.js` file instead of here?